src/components/sequencing/form/SelectedCommand.svelte

```diff
@@ -186,64 +175,84 @@
     return hasAncestorWithId(element.parentElement, id);
   }
 
-  // When the type in the argument value is compatible with the argument definition,
-  // provide a more restrictive editor to keep argument valid. Otherwise fall back on a string editor.
-
-  // TODO
-  // better handling of unclosed strings
+  function formatTypeName(s: string) {
+    // add spaces to CamelCase names, 'GroundEvent' -> 'Ground Event'
+    return s.replace(/([^A-Z])(?=[A-Z])/g, '$1 ');
+  }
 
-  // {'integer_arg', 'float_arg', 'unsigned_arg', 'enum_arg', 'var_string_arg', 'repeat_arg'}
-  // {'unsigned_arg', 'enum_arg', 'var_string_arg', 'float_arg'}
+  const nameArgumentDef: FswCommandArgumentVarString = {
+    arg_type: 'var_string',
+    default_value: null,
+    description: '',
+    max_bit_length: null,
+    name: '',
+    prefix_bit_length: null,
+    valid_regex: null,
+  };
 </script>
 
 <Panel overflowYBody="hidden" padBody={false}>
   <svelte:fragment slot="header">
-    <SectionTitle>Selected Command</SectionTitle>
+    <SectionTitle>{commandNode ? `Selected ${formatTypeName(commandNode.name)}` : ''}</SectionTitle>
   </svelte:fragment>
-
   <svelte:fragment slot="body">
-    <div id={ID_COMMAND_DETAIL_PANE} class="content">
-      {#if !!commandNode}
-        <div class="header"></div>
-
-        {#if !!commandDef}
-          {#if !!timeTagNode}
-            <fieldset>
-              <label for="timeTag">Time Tag</label>
-              <input class="st-input w-100" disabled name="timeTag" value={timeTagNode.text.trim()} />
-            </fieldset>
-          {/if}
+    <div id={ID_COMMAND_DETAIL_PANE} class="content"></div>
```

Comment:
What is the purpose of this div? Seems empty?

Reply:
Fixed a bad merge, I put that there to simplify how the debounced change listeners on the editor and form panel interact